examples/classification/plot_nb_comparison.py

```diff
@@ -0,0 +1,71 @@
+"""
+===========================================================
```

Comment:
Can't you just add to `examples/text/document_classification_20newsgroups.py`? This code doesn't seem to illustrate anything more specific to MNB/CMNB.